src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -1531,33 +1573,33 @@ void KaleidoScope_UpdateNamePanel(PlayState* play) {
     }
 }
 
-void func_808237B4(PlayState* play, Input* input) {
+void KaleidoScope_UpdateSwitchPage(PlayState* play, Input* input) {
     PauseContext* pauseCtx = &play->pauseCtx;
-    s32 cond = false;
-    s32 mode;
+    s32 frameAdvanceFreeze = false;
+    s32 nextPageMode;
 
-    if (ZREG(13) && !CHECK_BTN_ALL(input->press.button, BTN_L)) {
-        cond = true;
+    if (R_PAUSE_SWITCH_PAGE_FRAME_ADVANCE_ON && !CHECK_BTN_ALL(input->press.button, BTN_L)) {
```

Comment:
fwiw PageSwitch sounds better to me than SwitchPage in general, and it seems youve used that phrasing for other things such as `sPageSwitchEyeDx`
I guess this would apply to alot of things like `switchPageTimer`